pubspec.lock

```diff
@@ -222,7 +222,7 @@ packages:
     source: hosted
     version: "1.8.0"
   pedantic:
-    dependency: "direct main"
+    dependency: transitive
```

Comment:
How does this change relate to the issue being fixed?

Reply:
It doesn't, but unfortunately some of these files have a life of their own (I don't truly understand what causes these to change). 

I manually changed it back in